doc/source/user_guide/advanced.rst

```diff
@@ -965,21 +965,29 @@ If you select a label *contained* within an interval, this will also select the
    df.loc[2.5]
    df.loc[[2.5, 3.5]]
 
-``Interval`` and ``IntervalIndex`` are used by ``cut`` and ``qcut``:
+:func:`cut` and :func:`qcut` both return a ``Categorical`` object, and the bins they
+create are stored as an ``IntervalIndex`` in its ``.categories`` attribute.
 
 .. ipython:: python
 
    c = pd.cut(range(4), bins=2)
    c
    c.categories
 
-Furthermore, ``IntervalIndex`` allows one to bin *other* data with these same
-bins, with ``NaN`` representing a missing value similar to other dtypes.
+:func:`cut` also accepts an ``IntervalIndex`` for its ``bins`` argument, which enables
+a useful pandas idiom. First, We call :func:`cut` with some data and ``bins`` set to a
+fixed number, to establish the bins. Then, we can pass the value of ``.categories`` as
+the ``bins`` argument in subsequent calls to :func:`cut`, supplying new data which
+will be binned into the same bins.
 
 .. ipython:: python
 
    pd.cut([0, 3, 5, 1], bins=c.categories)
 
+When :func:`cut`  is passed an ``IntervalIndex`` for the `bins` argument, The ``Interval``
+values in the ``IntervalIndex`` define the bins and any value which falls outside all
+bins will be assigned a ``NaN`` value.
```

Comment:
I would keep it to the last part here (the former is already explained before the example), like "Values that fall outside the passed Intervals will be assigned a `NaN` value".

Reply:
You're right. Updated.